### DISCUSSION TITLE
CTk Label text in the middle of an image where I didn't code it to go!?

### DISCUSSION THREAD

**Discussion by Intiiii** [TomSchimansky/CustomTkinter#2329] [Q&A]
Hi, i've recently discovered customTkinter (like 3 hours ago (its 3 am :P)) and I'm atm changing all my code from Tkinter to CTk cuz its way better, but somehow there is a text in the middle of my background image!? 
![image](https://github.com/TomSchimansky/CustomTkinter/assets/81577133/167b9f58-f83f-41c9-a8c0-682fa90ab28e)
can someone help me?

**Comment by dipeshSam** [comment]
@IMT07 Pass `text=""` in your class initialiser as `CTkLabel(text="")`.
